test/test_bit_array_file.rb

```diff
@@ -0,0 +1,65 @@
+require "minitest/autorun"
```

Comment:
Note that I'm more familiar with rspec. Please excuse me if these tests aren't how most people would write `Minitest` tests.

Reply:
In a way, Minitest is to RSpec as Sinatra is to Rails. Minitest is far less opinionated and freeform which has upsides and downsides, but a big upside is pretty much any approach that works and fits into a project's own style is all good! :)